src/main/java/io/github/lambdatest/constants/Constants.java

```diff
@@ -7,15 +7,22 @@ public interface Constants {
 
   //SmartUI API routes
   interface SmartUIRoutes {
+    public static final String STAGE_URL = "https://stage-api.lambdatestinternal.com/visualui/1.0";
```

Comment:
Change the Key `STAGE_URL` to `HOST_URL`
and value to `https://api.lambdatest.com/visualui/1.0`